components/inputs/test/input-text.perf.js

```diff
@@ -1,24 +1,29 @@
 import '../input-text.js';
-import { expect, html } from '@open-wc/testing';
+import { html, testRenderTime } from 'web-test-runner-performance/browser.js';
```

Comment:
Curious what the differences are between the two `html`s? i.e. can we keep using the `@open-wc` one or do we need to switch for this to work?

Reply:
Sorry meant to leave a comment about this, internally this package [exposes lit](https://github.com/coryrylan/web-test-runner-performance/blob/main/src/browser.ts#L4). I first attempted with the `open-wc` one but I took a look at the tests rendering in non-headless mode and everything was getting rendered to the screen as `[object]`